tests/BrandingClientTest.php

```diff
@@ -153,16 +153,23 @@ public function testMalformedContentThrowsException()
     /**
      * @dataProvider cachingTimesDataProvider
      */
-    public function testCachingTimes($options, $headers, $expectedCacheDuration)
+    public function testCachingTimes2($options, $headers, $expectedCacheDuration)
     {
         $client = $this->getClient([$this->mockSuccessfulJsonResponse($headers)]);
-
-        $cache = $this->createPartialMock('Doctrine\Common\Cache\ArrayCache', ['save']);
-        $cache->expects($this->once())->method('save')->with(
-            $this->anything(),
-            $this->anything(),
-            $this->equalTo($expectedCacheDuration)
-        );
+        $cache = $this->getMockBuilder('Symfony\Component\Cache\Adapter\NullAdapter')
```

Comment:
Create partial mock disable the original constructor, which is necessary because the original has a closure in order to create a cacheItem. This is the reason why with createPartialMock the tests fail. I couldn't find any other function to construct the mock object allowing the original constructor.